src/clusterfuzz/_internal/google_cloud_utils/batch.py

```diff
@@ -281,10 +276,6 @@ def _get_spec_from_config(command, job_name):
     config_name += '-PREEMPTIBLE'
   else:
     config_name += '-NONPREEMPTIBLE'
-  # TODO(metzman): Get rid of this when we stop doing privileged operations in
```

Comment:
This cannot land before https://team-review.git.corp.google.com/c/gosst/clusterfuzz-config/+/2350609 which removes the UNPRIVILEGED suffix from the unprivileged config, and gets rid of the privileged configs.